api/observations.js

```diff
@@ -54,11 +54,12 @@ const getObservations = async () => {
     body.items.forEach((item) => {
       item.readings.forEach((reading) => {
         if (!reading.value) return;
+        const roundedValue = Math.round(reading.value * 10) / 10;
```

Comment:
I personally would do `Number(reading.value.toFixed(1))` instead for better readability.

Reply:
Updated!